packages/notebook/src/browser/notebook-output-utils.ts

```diff
@@ -0,0 +1,118 @@
+// *****************************************************************************
+// Copyright (C) 2023 TypeFox and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
```

Comment:
@tsmaeder @marcdumais-work : Could you comment?

Reply:
My info may not be quite up-to-date, but one option that was on the table to avoid piece-meal IP Check tickets for code copied or used from vscode was to have the IP team analyse the whole of specific versions of vscode \[1\]. e.g. start with latest published. Then we can use that version for such PRs, and as well for Theia Monaco update and when we build/publish new Theia vscode built-in extensions, all of these without the need for further IP Tickets.

Has something like the above been put in place?

\[1\] This might be a bit more work for the IP team the first time, but subsequently only differences could have to be analysed, lowering the work load.

Reply:
> You need to specify the commit from which this file was copied. We might get away without an IP check ticket if we pick the right one (I believe). See above.

Also ideally specify from which vscode file(s) the code was copied. Even if we have an IP Ticket that documents this, it will be easier to maintain if we have this info as comments in the sources.

Reply:
There used to be a policy whereby you could copy correctly licensed code as long as it did not exceed a certain size. Not sure where this might be documented, though.

Reply:
perfect! thanks alot